ci/terraform/oidc/mfa-reset-authorize.tf

```diff
@@ -22,9 +23,14 @@ module "mfa_reset_authorize" {
   environment     = var.environment
 
   handler_environment_variables = {
+    ENVIRONMENT                               = var.environment,
     IPV_AUDIENCE                              = var.ipv_audience
     MFA_RESET_STORAGE_TOKEN_SIGNING_KEY_ALIAS = aws_kms_alias.mfa_reset_token_signing_key_alias.arn,
     MFA_RESET_JAR_SIGNING_KEY_ALIAS           = aws_kms_alias.mfa_reset_jar_signing_key_alias.arn,
+    IPV_AUTHORISATION_CLIENT_ID               = var.ipv_authorisation_client_id
+    AUTH_ISSUER_CLAIM                         = "https://${local.frontend_fqdn}/",
+    IPV_AUTHORIZATION_URI                     = var.ipv_authorisation_callback_uri,
+    MFA_RESET_CALLBACK_URI                    = "https://${local.frontend_api_base_url}/reverification-callback"
```

Comment:
Nope the order is mainly a product of me forgetting some environment variables 😅. I think it would be neater if these were alphabetised though, so will do that!